android/java/org/chromium/chrome/browser/onboarding/OnboardingPrefManager.java

```diff
@@ -159,6 +160,10 @@ public void setNewOnboardingShown(boolean isShown) {
         sharedPreferencesEditor.apply();
     }
 
+    public boolean isBraveRewardsEnabled() {
+        return mSharedPreferences.getBoolean(BraveRewardsPreferences.PREF_ADS_SWITCH, false);
```

Comment:
Should it be isBraveAdsEnabled as function name ? as we are checking f or ADS_SWITCH pref